packages/react-components/src/useContractSelector.ts

```diff
@@ -101,7 +101,7 @@ export interface ContractSelector {
 
 /**
  * React hook to look up a smart contract's data and state from its index.
- * @param rpc JSON-RPC proxy client through which to perform the lookup.
+ * @param rpc gRPC client through which to perform the lookup. The JSON-RPC client is supported for backwards compatibility only.
```

Comment:
Support for `JsonRpcClient` will be removed once we upgrade to a version of `web-sdk` that doesn't include the type.